dht/token.hh

```diff
@@ -98,10 +98,15 @@ public:
     }
 
     bytes data() const {
-        auto t = net::hton(_data);
-        bytes b(bytes::initialized_later(), sizeof(_data));
-        std::copy_n(reinterpret_cast<int8_t*>(&t), sizeof(_data), b.begin());
-        return b;
+        return write_bytes_be(_data);
+    }
+
+    bytes unsigned_data() const {
```

Comment:
This function needs a comment why it exists. It's not at all obvious what does "unsigned data" mean. To be honest, it's also not clear what data() means - it should have had a comment too...